scripts/StateManager.js

```diff
@@ -25,17 +26,28 @@ function StateManager() {
 	this.activate = function() {
 		isKKTime = timeKeeper.getDay() == 6 && timeKeeper.getHour() >= 20;
 		getSyncedOptions(function() {
+		if(!weatherManager) {
+			weatherManager = new WeatherManager(options.zipCode, options.countryCode);
+			weatherManager.registerChangeCallback(function() {
+				if(!isKK() && options.music == 'new-leaf-live') {
```

Comment:
You may want to change "options.music == 'new-leaf-live'" to "isLive()" so you won't have to change this conditional if we add more live weather music options.